packages/main/src/TableRowBase.ts

```diff
@@ -121,3 +122,9 @@ abstract class TableRowBase extends UI5Element {
 }
 
 export default TableRowBase;
+
+const isInstanceOfTableRowBase = (obj: any) => obj instanceof HTMLElement && obj.hasAttribute("ui5-table-row-base");
```

Comment:
why do we need those methods? the idea was getting rid of such methods. this will probably help less import in other components.

If you are insist, somehow to align with other web components, then still I would expect that every class add their own check, not the base.